static/app/views/organizationGroupDetails/groupReplays/groupReplays.spec.tsx

```diff
@@ -0,0 +1,460 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen, userEvent, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import ProjectsStore from 'sentry/stores/projectsStore';
+import {OrganizationContext} from 'sentry/views/organizationContext';
+import GroupReplays from 'sentry/views/organizationGroupDetails/groupReplays';
+import {RouteContext} from 'sentry/views/routeContext';
+
+type InitializeOrgProps = {
+  location?: {
+    pathname?: string;
+    query?: {[key: string]: string};
+  };
+  organizationProps?: {
+    features?: string[];
+  };
+};
+
+const mockUrl = '/organizations/org-slug/replays/';
+
+const mockProps = {
+  group: TestStubs.Group(),
+  replayIds: ['346789a703f6454384f1de473b8b9fcc', 'b05dae9b6be54d21a4d5ad9f8f02b780'],
+};
+
+let mockRouterContext: {
+  childContextTypes?: any;
+  context?: any;
+} = {};
+
+const getComponent = ({
+  location,
+  organizationProps = {features: ['session-replay-ui']},
+}: InitializeOrgProps) => {
+  const {router, organization, routerContext} = initializeOrg({
+    organization: {
+      ...organizationProps,
+    },
+    project: TestStubs.Project(),
+    projects: [TestStubs.Project()],
+    router: {
+      location: {
+        pathname: '/organizations/org-slug/replays/',
+        query: {},
+        ...location,
+      },
+    },
+  });
+
+  ProjectsStore.init();
+  ProjectsStore.loadInitialData(organization.projects);
+
+  mockRouterContext = routerContext;
+
+  return (
+    <OrganizationContext.Provider value={organization}>
+      <RouteContext.Provider
+        value={{
+          router,
+          location: router.location,
+          params: router.params,
+          routes: [],
+        }}
+      >
+        <GroupReplays {...mockProps} />
+      </RouteContext.Provider>
+    </OrganizationContext.Provider>
+  );
+};
+
+const renderComponent = (componentProps: InitializeOrgProps = {}) => {
+  return render(getComponent(componentProps), {context: mockRouterContext});
+};
+
+describe('GroupReplays', () => {
+  beforeEach(() => {
+    MockApiClient.clearMockResponses();
+  });
+
+  // Assert that query to the events endpoint is correct
+  it('Should have correct queries in the events endpoint', async () => {
```

Comment:
seems like the comment is trying to help describe the test. Instead of using a comment, can we improve the name of the test?

```suggestion
  it('should query the events endpoint with the passed in replayIds', async () => {
```

something like that imo